server/viame_tasks/tasks.py

```diff
@@ -150,3 +167,50 @@ def convert_video(self, path, folderId, token, auxiliaryFolderId):
         _file["itemId"], {"folderId": folderId, "codec": "h264"}
     )
     os.remove(output_path)
+
+
+@app.task(bind=True)
+def convert_images(self, folderId, token):
+    """
+    Ensures that all images in a folder are in a web friendly format (png or jpeg).
+
+    If conversions succeeds for an image, it will replace the image with an image
+    of the same name, but in a web friendly extension.
+
+    Returns the number of images successfully converted.
+    """
+    gc = self.girder_client
+    gc.token = token
+
+    items = gc.listItem(folderId)
+    skip_item = (
+        lambda item: item["name"].endswith(".png")
+        or item["name"].endswith(".jpeg")
+        or item["name"].endswith(".jpg")
+    )
+    valid_items = [item for item in items if not skip_item(item)]
```

Comment:
Rename to `items_needing_conversion` or something like that.  `valid` is ambiguous.